.github/workflows/run-terraform.yml

```diff
@@ -88,6 +88,11 @@ on:
         required: false
         type: boolean
         default: false
+      unlock:
+        description: "An optional string which runs terraform force-unlock on the provided `LOCK_ID`, if set."
+        required: false
+        type: string
+        default: ""
```

Comment:
I started out using a boolean, but it seemed redundant as you'll always need the TF state ID. This way you just provide `unlock: ID000` to unlock your state. Instead of `unlock: true` `state_id: ID000`.